polyfills/Object.prototype.toString/polyfill.js

```diff
@@ -0,0 +1,6 @@
+var Object_prototype_toString = Object.prototype.toString
+Object.prototype.toString = function() {
+    if (this === null) return "[object Null]";
```

Comment:
I don't believe this will work as per 15.3.4.4 from the ES3 specification:
>Function.prototype.call (thisArg [ , arg1 [ , arg2, … ] ] )
>...
>If thisArg is null or undefined, the called function is passed the global object as the this value. Otherwise, the called function is passed ToObject(thisArg) as the this value.